### DISCUSSION TITLE
Why CandlePart was discarded in getting EMA?

### DISCUSSION THREAD

**Discussion by tiger2014** [DaveSkender/Stock.Indicators#963] [Q&A] [answered]
I found out that in the latest release, CandlePart.Close is invalid when we get the EMA. Why do we discard it?

**Comment by DaveSkender** [comment, marked as answer]
With version 2, we added a more comprehensive chaining model, so the same can be achieved with (example):

``` csharp
var results = quotes
  .Use(CandlePart.HL2)
  .GetEma(..);
```
There are also more candle part options now.  See [utilities and helpers](https://dotnet.stockindicators.dev/utilities/#use-alternate-price) for more information.

**Comment by tiger2014** [comment]
Thanks!
